op-service/sources/types.go

```diff
@@ -61,6 +61,9 @@ type RPCHeader struct {
 	// ParentBeaconRoot was added by EIP-4788 and is ignored in legacy headers.
 	ParentBeaconRoot *common.Hash `json:"parentBeaconBlockRoot,omitempty"`
 
+	// RequestsHash was added by EIP-7685 and is ignored in legacy headers.
```

Comment:
I just took a look to see if this could be validated in some way but it doesn't look like we can access the requests from the EL

Reply:
Request entries don't persist in the EL unfortunately. Once exported during block-building they become a consensus-layer-only thing. Technically we can reconstruct them from receipts, but that's not needed here.